lib/k8s-e2e-tests/tests/vector-agent.rs

```diff
@@ -84,110 +74,6 @@ const CUSTOM_RESOURCE_VECTOR_CONFIG: &str = indoc! {r#"
             encoding = "json"
 "#};
 
-/// This test validates that vector-agent picks up logs at the simplest case
-/// possible - a new pod is deployed and prints to stdout, and we assert that
-/// vector picks that up.
```

Comment:
Just to check, does the `ct` test actually verify that Vector picks up logs? Or just that it deploys correctly?

Reply:
Just that it deploys correctly - there's still a test in integrations that tests the "simple" case, we just don't do it multiple times with the different ways we currently have to render config

Reply:
Ah, gotcha 👍 That was going to be my next question, if we were testing that elsewhere.